src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/OpticalDuplicateFinder.java

```diff
@@ -185,6 +188,14 @@ protected final int rapidParseInt(final String input) {
         return val;
     }
 
+    public boolean isOpticalDuplicate(PhysicalLocation a, PhysicalLocation b) {
+      return a.getReadGroup() == b.getReadGroup() &&
+          a.getTile() == b.getTile() &&
+          // There used to not be an abs here, not sure why.
```

Comment:
remove this comment. Either the abs is ok and it was a bug before or the abs is not ok and you should remove it. can you figure out which one is it?